apps/Smart/Radarr/code.js

```diff
@@ -1,20 +1,32 @@
-﻿class Radarr
-{ 
-    getUrl(args, endpoint){
+﻿class Radarr {
+    getUrl(args, endpoint) {
         return `api/v3/${endpoint}?apikey=${args.properties['apikey']}`;
     }
-    
+
     async status(args) {
+        let filter = args.properties['filters'];
+
         let data = await args.fetch(this.getUrl(args, 'movie'));
-        let missing = data?.filter(x => x.hasFile == false)?.length ?? 0;
+        let filteredData = data?.filter((x, index, arr) => {
```

Comment:
need an if check here for show all missing.   I would make none value "", so upgrades you can just do 
if(filter) {
... new code
}
and not have it effect existing users

Reply:
So it still runs normal filter of 
x.hasFile == false;

And would just skip over the other 2 if statements

Reply:
Do you mean like this? (with all value also set to '' ) 
I am not sure if anything is gained from adding this but happy to do so?
![image](https://user-images.githubusercontent.com/77850077/156942029-a3a5d6a6-2c9a-4d7d-9784-88d81ab07989.png)

Reply:
nah i just missed you were setting it valid by default and only filtering after.